packages/typespec-vscode/package.json

```diff
@@ -122,13 +122,23 @@
           "command": "typespec.generateCode",
           "when": "explorerResourceIsFolder || resourceLangId == typespec",
           "group": "typespec@1"
+        },
+        {
+          "command": "typespec.showOpenApi3",
+          "when": "resourceLangId == typespec",
+          "group": "typespec@1"
```

Comment:
the @# is for ordering. I think this will be used more than import by end user. Please change this to @2 and the import above to @3

Reply:
updated the ordering